eZ/Publish/API/Repository/Tests/FieldType/CheckboxIntegrationTest.php

```diff
@@ -392,6 +387,21 @@ public function testFindContentFieldCriterion(bool $isActive): void
         self::assertSame($isActive, $value->bool);
     }
 
+    public function testAddFieldDefinition()
+    {
+        $content = $this->addFieldDefinition();
+
+        $this->assertCount(2, $content->getFields());
```

Comment:
Static methods should be called statically.
```suggestion
        self::assertCount(2, $content->getFields());
```